CHANGELOG.asciidoc

```diff
@@ -25,6 +25,10 @@ https://github.com/elastic/apm-agent-python/compare/v5.3.0\...master[Check the d
 // Unreleased changes go here
 // When the next release happens, nest these changes under the "Python Agent version 5.x" heading
 
+===== New Features
```

Comment:
@bmorelli25 the docs build is failing: 
```
22:09:16 INFO:build_docs:asciidoctor: WARNING: ../CHANGELOG.asciidoc: line 28: section title out of sequence: expected level 3, got level 4
```
However, I don't want to just make it a level three heading, or else we'll have to modify all the heading levels every time we release. Is there a way to tell it to ignore this issue? Alternatively, we could add a level 3 heading between `Unreleased` and `New Features`, though that seems silly. Thoughts?

Reply:
> Is there a way to tell it to ignore this issue

Yes! This needs a float tag: `[float]`. This tells the build to "float" the heading to the current page and not include it in the section hierarchy. That will fix this error.

Reply:
As an example: https://github.com/elastic/apm-agent-python/blame/ebe1c4a6ca057cadac15ac6726527428967d1679/CHANGELOG.asciidoc#L13-L14

Reply:
Perfect. I'll make that change.